Resources/Prototypes/Entities/Markers/Spawners/Mobs/animals.yml

```
@@ -0,0 +1,273 @@
+- type: entity
+  name: Mouse Spawner
+  id: SpawnMobMouse
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    layers:
+      - state: green
+      - sprite: Mobs/Animals/mouse.rsi
+        state: icon-2
+      - state: ai
```

incorrect indentation as per conventions, ex. shown below
```suggestion
    layers:
    - state: green
    - sprite: Mobs/Animals/mouse.rsi
      state: icon-2
    - state: ai
```